src/main/java/org/myrobotlab/service/config/ServiceConfig.java

```diff
@@ -257,6 +255,10 @@ public static Plan getDefault(Plan plan, String name, String inType) {
         log.info("could not find config class for {}, loading generalized ServiceConfig", inType);
         ServiceConfig sc = new ServiceConfig();
         sc.type = inType;
+        if (name.equals("i01")) {
+          log.info("here");
+        }
+        Collections.sort(sc.listeners, new MrlListenerComparator());
```

Comment:
why the need for the sort here?  it'd be nice to preserve the original order, wouldn't it?

Reply:
Considering the amount of some service's subscriptions its much easier to examine and maintain if they are sorted.
The way they were entered in has no impact at how subscriptions are processed.
Trying to diff a checked in config vs new unsorted is difficult